src/main/resources/application.yaml

```diff
@@ -139,6 +139,8 @@ launchDarkly:
   connectionTimeout: 5000
   socketTimeout: 5000
 
+environment: ${ENVIRONMENT:local}
```

Comment:
you need to set this env variable in chart file to read the values of the other environments. Otherwise it will always set to local.
Like this - ENVIRONMENT: "{{ .Values.global.environment }}"

Reply:
Good point. Added to values.yaml